tools/migrate_sql_to_json.php

```diff
@@ -0,0 +1,50 @@
+<?php
```

Comment:
please move into the `tools/single_use` directory

Reply:
is it single use, though? It will be used multiple times for different instruments.

Reply:
I mean it's not meant to be reused for the same instrument...